eng/pipelines/npm-tasks.yml

```diff
@@ -51,12 +51,40 @@ extends:
   parameters:
     stages:
       - stage:
-        displayName: NPM Task Execute
+        displayName: 'NPM Task Execute'
+        variables:
+          - template: /eng/pipelines/templates/variables/image.yml
 
         jobs:
-        - deployment: 'NPM_Admin'
-          displayName: NPM package management
+        - job: NPM_Admin
+          displayName: 'NPM package management'
+          pool:
+            name: $(WINDOWSPOOL)
+            image: $(WINDOWSVMIMAGE)
+            os: windows
+
+          steps:
+            - checkout: self
+            - download: current
+            - task: PowerShell@2
+              displayName: 'Run Task'
+              inputs:
+                targetType: filePath
+                filePath: "$(System.DefaultWorkingDirectory)/eng/scripts/npm-admin-tasks.ps1"
+                arguments: >
+                  -taskType ${{parameters.TaskType}}
+                  -packageName ${{parameters.PackageName}}
+                  -pkgVersion ${{parameters.PkgVersion}}
+                  -tagName ${{parameters.TagName}}
+                  -npmToken "$(azure-sdk-npm-token)"
+                  -reason "${{parameters.Reason}}"
+                pwsh: true
+
+        - deployment: NPMSuccess
+          displayName: NPM package management success
           environment: npm
+          dependsOn: NPM_Admin
+          condition: succeeded()
 
           pool:
```

Comment:
Can we use an agentless job here instead so we don't have to provision a VM just for an approval gate?

https://learn.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#server-jobs

Perhaps delay for 0? https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/delay-v1?view=azure-pipelines